models/pull.go

```diff
@@ -175,7 +175,11 @@ func (pr *PullRequest) GetDefaultMergeMessage() string {
 			return ""
 		}
 	}
-	return fmt.Sprintf("Merge branch '%s' of %s/%s into %s", pr.HeadBranch, pr.MustHeadUserName(), pr.HeadRepo.Name, pr.BaseBranch)
+	if err := pr.LoadIssue(); err != nil {
+		log.Error("Cannot load issue %d for PR id %d: Error: %v", pr.IssueID, pr.ID, err)
+		return ""
+	}
+	return fmt.Sprintf("Merge pull request %d: '%s' from '%s' of %s/%s into %s", pr.IssueID, pr.Issue.Title, pr.HeadBranch, pr.MustHeadUserName(), pr.HeadRepo.Name, pr.BaseBranch)
```

Comment:
```suggestion
	return fmt.Sprintf("Merge pull request '%s' (#%d) from %s/%s into %s", pr.Issue.Title, pr.IssueID, pr.MustHeadUserName(), pr.HeadBranch, pr.BaseBranch)
```
this is more like github + has still the PR title in it

Reply:
You need index not ID

Reply:
more :eyes: see more - added to suggestion

Reply:
@6543 I have made the change.